simple_history/management/commands/populate_history.py

```diff
@@ -54,14 +57,16 @@ def handle(self, *args, **options):
             for model in models.registered_models.values():
                 try:    # avoid issues with mutli-table inheritance
                     history_model = utils.get_history_model_for_model(model)
-                except utils.NotHistorical:
+                except NotHistoricalModelError:
                     continue
                 to_process.add((model, history_model))
             if not to_process:
-                self.stdout.write(self.NO_REGISTERED_MODELS)
+                if self.verbosity >= 1:
```

Comment:
Can we get constants for the verbosity levels? This is magic AF

Reply:
It's actually Django AF. https://github.com/django/django/blob/master/django/core/management/commands/makemigrations.py#L167